src/status_im2/subs/chat/messages.cljs

```diff
@@ -102,6 +102,25 @@
  (fn [messages]
    (empty? messages)))
 
+(defn albumize-messages
+  [messages]
+  (get (reduce (fn [{:keys [messages albums]} message]
+                 (let [album-id (when (:albumize? message) (:album-id message))
+                       albums   (cond-> albums album-id (update album-id #(conj % message)))
```

Comment:
@OmarBasem, using `update` like this with the anonymous function is redundant, because `update` already calls the function by passing the updatable value as the first argument. This is by design, very useful.

```clojure
;; Old
(update album-id #(conj % message))

;; New
(update album-id conj message)
```

```clojure
;; Proof
(assert [1 2 3 4]
        (update {"0x1" [1 2 3]} "0x1" conj 4))
```

Reply:
Thanks!